types.d.ts

```diff
@@ -0,0 +1,2 @@
+/** Experimental exports, shape of type exports may change in the future */
+export * from './typings/core/types/typegram'
```

Comment:
Is there any documentation about how this can be used, or how it should preferably be used? Not sure if docs are required, I'm just wondering how this will look

Reply:
No docs for now. As a first step, it allows consumers to use the same types Telegraf does, and I'd like to shape it better depending on how it's being used.